solr/core/src/java/org/apache/solr/handler/admin/CollectionsHandler.java

```diff
@@ -517,10 +517,7 @@ private static void addStatusToResponse(NamedList<Object> results, RequestStatus
           ColStatus.RAW_SIZE_DETAILS_PROP,
           ColStatus.RAW_SIZE_SAMPLING_PERCENT_PROP,
           ColStatus.SIZE_INFO_PROP);
-      // make sure we can get the name if there's "name" but not "collection"
-      if (props.containsKey(CoreAdminParams.NAME) && !props.containsKey(COLLECTION_PROP)) {
```

Comment:
Removed this because it's not documented in the API anyway and the CoreAdminParams.NAME isn't copied anyway.